helm/charts/determined/templates/genai/genai-deployment.yaml

```diff
@@ -32,21 +32,25 @@ spec:
           - name: DB_NAME
             value: lore
           - name: DB_USER
-            value: postgres
+            value: {{ .Values.db.user | quote }}
           - name: DB_PASSWORD
-            value: postgres
+            value: {{ .Values.db.password | quote }}
           - name: DB_PORT
-            value: "5432"
+            value: {{ .Values.db.port | quote }}
           - name: DB_HOST
             value: {{ include "determined.dbHost" . }}
           - name: LORE_DOCKER_TAG_SUFFIX
             value: {{ $tag }}
           - name: K8S_SHARED_PVC_NAME
             value: {{ include "genai.PVCName" . }}
+          - name: RESOURCE_POOL_EXTRA_METADATA_PATH
+            value: /run/determined/workdir/resource_pool_metadata.yaml
         volumeMounts:
           - name: genai-pvc-storage
             mountPath: /run/determined/workdir/shared_fs
             readOnly: false
+          - name: genai-resource-pool-metadata
+            mountPath: /run/determined/workdir
```

Comment:
I am always worried about mounts like that because you don't know if the previous shared_fs mount will be mounted over and become unavailable. does it work?

Reply:
It does work. I suppose something like this could be mounted else where? This just seemed like the right place to put it. Maybe some other sub directory specific for it would be best

Reply:
putting it in a separate directory `/run/determined/workdir/rp_config`